xc/xc7/tests/soc/litex/linux/soc_linux.py

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/env python3
+
+import os
+import json
+import subprocess
+
+from migen import ClockDomain
+
+from litex.soc.cores.gpio import GPIOOut, GPIOIn
+from litex.soc.cores.spi import SPIMaster
+from litex.soc.cores.bitbang import I2CMaster
+from litex.soc.cores.xadc import XADC
+from litex.soc.cores.pwm import PWM
+from litex.soc.cores.icap import ICAPBitstream
+from litex.soc.cores.clock import S7MMCM
+
+# from litevideo.output import VideoOut
+# from migen.build.generic_platform import Pins, IOStandard, Subsignal
```

Comment:
It's worth removing these if they are not needed

Reply:
I left them, because the idea of this PR is to gradually enable features which we do not support in SymbiFlow yet.
However, Framebuffer is something that we can just remove for now.